pest/src/parser.rs

```diff
@@ -8,12 +8,14 @@
 use std::rc::Rc;
 
 use super::error::Error;
-use super::inputs::Input;
+use super::inputs::{Input, StringInput};
 use super::iterators::Pairs;
 use super::RuleType;
 
 /// A `trait` that defines a `Parser`.
 pub trait Parser<R: RuleType> {
     /// Parses `input` starting from `rule`.
     fn parse<I: Input>(rule: R, input: Rc<I>) -> Result<Pairs<R, I>, Error<R, I>>;
+    /// Parses an `input` string starting from `rule`.
```

Comment:
This should specify that we're parsing a `&str`. It should also mention that it's a convenience method.